src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -31,11 +32,15 @@
 _NUM_RETRIES = 3
 _ISSUE_TRACKER_URL = 'https://issues.chromium.org/issues'
 
+# TODO: Make these configuration settings instead of hardcoded values in code.
 # These custom fields use repeated enums.
 _CHROMIUM_OS_CUSTOM_FIELD_ID = '1223084'
```

Comment:
As you are likely aware but making it explicit here- If there are any assumptions about these custom fields existing in the code then they will fail in the non-Chromium tracker. Doing a quick check it should be OK, but mentioning incase you see errors about that later.

Reply:
Ack yep -- there aren't overlaps currently between Chromium and OSS-Fuzz.